MAVProxy/modules/lib/live_graph_ui.py

```diff
@@ -66,7 +66,7 @@ def init_plot(self):
             self.axes.set_facecolor('white')
         except AttributeError as e:
             # this was removed in matplotlib 2.2.0:
-            self.axes.set_axis_bgcolor('white')
```

Comment:
are you sure this is right? We only get to this line if calling set_facecolor() caused an AttributeError exception